packages/vite/src/node/plugins/workerImportMetaUrl.ts

```diff
@@ -0,0 +1,95 @@
+import type { ResolvedConfig } from '../config'
+import type { Plugin } from '../plugin'
+import { getAssetHash, fileToUrl } from './asset'
+import {
+  cleanUrl,
+  injectQuery,
+  multilineCommentsRE,
+  singlelineCommentsRE
+} from '../utils'
+import path from 'path'
+import { parseWorkerRequest, bundleWorkerEntry } from './worker'
+import { ENV_PUBLIC_PATH } from '../constants'
+import MagicString from 'magic-string'
+
+const WorkerFileId = 'worker_url_file'
```

Comment:
IMO this doesn't follow any kind of known convention, so either we should write

```suggestion
const workerFileId = 'worker_url_file'
```

or

```suggestion
const WORKER_FILE_ID = 'worker_url_file'
```